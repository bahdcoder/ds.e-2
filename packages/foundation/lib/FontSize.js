const fontSizes = {
    xs: 'xs',
    sm: 'sm',
    base: 'base',
    lg: 'lg',
    xl: 'xl',
};
export default Object.freeze(fontSizes);
