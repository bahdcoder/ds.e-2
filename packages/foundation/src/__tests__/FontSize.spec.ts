import FontSize from '../FontSize'

test('should match snapshot', () => {
    expect(FontSize).toMatchSnapshot()
})
