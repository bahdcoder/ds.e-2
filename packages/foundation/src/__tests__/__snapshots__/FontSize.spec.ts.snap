// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "base": "base",
  "lg": "lg",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
}
`;
